pkgs/servers/fiche/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "fiche";
+  version = "0.9.1";
+
+  src = fetchurl {
+    url = "https://github.com/solusipse/${pname}/archive/${version}.tar.gz";
+    sha256 = "TODO";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv fiche $out/bin
```

Comment:
You can use install here to also set the permission correct.

Reply:
Could you elaborate? I don't understand what I have to change.

Reply:
see https://github.com/solusipse/fiche/blob/master/Makefile#L9

Reply:
So just replace those two lines with this?
```
install -m 0755 fiche $out
```

Reply:
I have pushed a commit with the suggested changes